pages/sitemap.ts

```diff
@@ -0,0 +1,18 @@
+import { MetadataRoute } from 'next';
+
+export default function sitemap(): MetadataRoute.Sitemap {
+  const baseUrl: string | undefined = process.env.NEXT_PUBLIC_WEBSITE_URL;
```

Comment:
It will be better to check the environment variable and throw exception and then create the baseUrl variable